privaterelay/templates/includes/alias-card.html

```diff
@@ -2,20 +2,22 @@
 {% ftlconf bundle='privaterelay.ftl_bundles.main' %}
 {% load static %}
 
-<!-- 
+{% comment %}
```

Comment:
Good looking out!

Reply:
I was inspecting code and saw the same big 'ol comment block for every. single. aliases. 🙃